src/pipeline/testing_pipeline.py

```diff
@@ -40,11 +40,14 @@ def _create_saving_dir(self, args):
     def _setup_config(self):
         pass
 
+    def _setup_optimizers(self):
+        return []
```

Comment:
Do we need this function in the testing pipeline?

Reply:
Fixed